### DISCUSSION TITLE
Clothing when worn does not display the inventory slot

### DISCUSSION THREAD

**Discussion by cccpvs** [FakeFishGames/Barotrauma#9161] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

If you create an **ItemContainer** in clothes, then when wearing these clothes, what is in the cell is not displayed.

When using the _hideitems="false"_ parameter in the **ItemContainer**, this only works when the item is on the ground, but not when worn on the player.

![](https://barotrauma.dayz-vologda.ru/images/Issue_bug_Wearable_itemContainer.png)

### Bug prevalence

Happens regularly

### Version

0.17.15.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I'd say this is a feature request for adding support for displaying the contained items on the clothing, not a bug report, so I'll move this to discussions.
